core/tests/tests/Math.mint

```diff
@@ -80,6 +80,12 @@ suite "Math.fmod" {
   }
 }
 
+suite "Math.truncate" {
+  test "it truncates number to given amount" {
```

Comment:
Copy pasted from `Math.truncate`.

Reply:
I don't understand your comment :) please elaborate.

Reply:
It's copy pasted from `Math.truncate` specs - which is a different method.

Reply:
Ah, I see now the name should be `Math.trunc`.